db/migrate/20241224140952_add_description_to_notes.rb

```diff
@@ -0,0 +1,19 @@
+class AddDescriptionToNotes < ActiveRecord::Migration[7.2]
+  def change
+    add_column :notes, :description, :text, :null => false, :default => ""
+    add_column :notes, :user_id, :bigint
+    add_column :notes, :user_ip, :inet
+
+    Note.find_each do |note|
+      first_comment = note.all_comments.first
+      if first_comment
+        note.update(
+          :description => first_comment.body,
+          :user_id => first_comment.author_id,
+          :user_ip => first_comment.author_ip
+        )
+        first_comment.update(:body => "")
+      end
+    end
+  end
+end
```

Comment:
We should avoid running backfill directly in the migration as it risks downtime with large datasets. We should move this to a Rake task or background job for safer execution.

Also few more thinks to consider:
- Should we remove first comment once we backfill the body here, or make a new migration task...
- Should we consider doing this in batches or try to do this all at once
- Should we include some kind of fault-tolerance mechanism - if execution fails, will it try to retry and be able to continue where it left off